RadixWallet/Core/SharedModels/Assets/OnLedgerEntity.swift

```diff
@@ -137,7 +137,7 @@ extension OnLedgerEntity {
 		public let poolUnit: PoolAddress?
 		public let poolUnitResource: ResourceAddress?
 		public let claimedEntities: [String]?
-		public let claimedWebsites: [URL]?
+		public var claimedWebsites: [URL]?
```

Comment:
This is not meant to be mutable, I see the reason the change was made, but it is preferable to not change the domain model to accommodate a specific UI need